### PR TITLE
minor fix in description of self-employed input variable

### PR DESCRIPTION
**Pull request by ChristianZimpelmann**
Minor fixes in description of input_variables

### CODE REVIEW

File: docs/gettsim_objects/input_variables.rst [hunk 1]
```diff
@@ -32,9 +32,9 @@ household.
 +---------------------------+---------------------------------------------+--------------+
 | _`in_ausbildung`          | Dummy: Employment status “in education”     | bool         |
 +---------------------------+---------------------------------------------+--------------+
-| _`selbstständig`          | Dummy: Individual is self-employed          | bool         |
+| _`selbstständig`          | Dummy: Full-time self-employed              | bool         |
```
Comment (hmgaudecker) [inline docs/gettsim_objects/input_variables.rst:35]: Why would it have to be full time? I guess it is more about the main activity? Will need to rethink that classification at some point, anyhow.

Comment (ChristianZimpelmann) [reply docs/gettsim_objects/input_variables.rst:35]: You are right. We need the translation of "hauptberuflich". 

How about `Dummy: Self-employed (main profession)` ? Alternatively, we can just add the German term

Comment (hmgaudecker) [reply docs/gettsim_objects/input_variables.rst:35]: Just use that, perfectly fine. 

Before a 1.0 release we should definitely go for a categorical variable with the main type of activity (employment subject to social security contributions, civil servant, self-employed, unemployed, out of the labor force, parental leave, pensioner,  ...). Will get hairy with pensioner / employment definition from 2023 onwards, of course... Don't we already have an issue for that?

Comment (ChristianZimpelmann) [reply docs/gettsim_objects/input_variables.rst]: 👍 

I don't remember and haven't found one.

**Comment by LauraGergeleit** [comment]
Looks fine to me!

**Review by LauraGergeleit — APPROVED** [review]
Looks fine to me!

**Pull request merged**